docs/static/releasenotes.asciidoc

```diff
@@ -65,6 +66,75 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
 
+[[logstash-7-17-6]]
+=== Logstash 7.17.6 Release Notes
+
+[[notable-7-17-6]]
+==== Notable issues fixed
+
+* Support Ironbank Docker build https://github.com/elastic/logstash/pull/14298[#14298]
```

Comment:
This isn't really a "notable issue fixed" - its related to our build process, and how we build the DoD docker images.